docs/imviz/plugins.rst

```diff
@@ -74,6 +74,23 @@ parameters, shape, and orientation (if applicable) all update concurrently.
 
 Angle is counter-clockwise rotation around the center in degrees.
 
+.. _markers-plugin:
+
+Markers
+=======
+
+This plugin allows for interactively creating markers in any viewer and logging information about
+the location of that marker along with the applicable data and viewer labels into a table.
+
+With the plugin open in the tray, mouse over any viewer and press the "m" key to log the information
+displayed in the app toolbar into the table.  The markers remain at that fixed pixel-position in
+the viewer they were created (regardless of changes to the underlying data or linking) and are only
+visible when the plugin is opened.
+
+To export the table into the notebook via the API, call
```

Comment:
Ori made a section here for exporting data. Should we move this info over to https://jdaviz.readthedocs.io/en/latest/imviz/export_data.html ?

Reply:
Also not sure... this is not imviz-specific so definitely shouldn't be _moved_ there (but we could either copy the info or link back to here).  Does it count as data?  Do we want to have every plugin that supports exporting something mentioned there or maybe as more plugins adopt tables we can have something more general somewhere else?

Reply:
> Does it count as data?

Yes, because aperture photometry table is over there.

> Do we want to have every plugin that supports exporting something mentioned there or maybe as more plugins adopt tables we can have something more general somewhere else?

This is better answered by @orifox or @camipacifici .

Reply:
ok, for now I'll add a link from that page back to the plugin docs and as we add more plugins with tables, we can consider moving general examples to a better shared place in the docs.